tests/config/playwright.config.ts

```diff
@@ -20,26 +21,27 @@ const testDir = path.join(__dirname, "..", "..", "apps/web/playwright");
 const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
   retries: 1,
-  workers: 1,
-  timeout: 60_000,
+  timeout: 80000,
+  workers: !!process.env.CI || process.env.PLAYWRIGHT_HEADLESS === "1" ? os.cpus().length : 1,
   reporter: [
-    [process.env.CI ? "github" : "list"],
+    [process.env.CI ? "list" : "list"], // THis won't be part of the PR. It is added for testing purposes.
     ["html", { outputFolder: "./playwright/reports/playwright-html-report", open: "never" }],
     ["junit", { outputFile: "./playwright/reports/results.xml" }],
   ],
   globalSetup: require.resolve("./globalSetup"),
   outputDir,
   webServer: {
-    command: "NEXT_PUBLIC_IS_E2E=1 yarn workspace @calcom/web start -p 3000",
+    command:
+      "NEXT_PUBLIC_IS_E2E=1 NEXT_PUBLIC_LICENSE_CONSENT=agree yarn workspace @calcom/web start -p 3000",
```

Comment:
Why don't we instead add an exception for E2E in here? https://github.com/calcom/cal.com/blob/a3e0d0aec9b01294314fe1cd42f2449ffe739512/apps/web/ee/components/LicenseBanner.tsx#L23

Reply:
I recently added this to understand what is going on because it is working fine in my own repo (forked not the local copy)  but fails in Cal's account. If you check the log for the workflow you will see that NEXT_PUBLIC_LICENSE_CONSENT is missing despite I have added in e2e.yml. So the cache Node uses is not being invalidated for some reason. I'm not a GH actions wizard so I might don't understand well why changes in .github/<workflow> are not taken into account when I pushed running my PR. But, all in all, what I'm trying to say is that this particular addition was a temporary hotfix to force the setting of this env var I'm using. It is not planned to be within prod code.

Seems that the banner of the LICENSE consent forces an unwanted autoscrolling within the playwright job that prevents the test for clicking the submit button when saving a new schedule. It is a minor issue solved with setting ON this env var but I'm struggleling to make it work only in Cal's account.

Reply:
Fine by me 👍🏽